code/controllers/stonedmc/master.dm

```diff
@@ -176,7 +176,7 @@ GLOBAL_REAL(Master, /datum/controller/master) = new
 	if(init_sss)
 		init_subtypes(/datum/controller/subsystem, subsystems)
 
-	to_chat(world, "<span class='boldnotice'>Initializing subsystems...</span>")
+	to_chat(world, "<span class='boldannounce'>Initializing subsystems...</span>")
```

Comment:
Keep these blue please.

Reply:
Why were they changed from red in the first place?

Reply:
Aesthetics. Red is scary.

Reply:
red because tg mc always print out red texts